resources/views/create.blade.php

```diff
@@ -1,34 +1,31 @@
 @extends('layouts.app')
 
 @section('content')
-    <div class="container">
-        <div class="row">
-            <div class="col-md-10 col-md-offset-1">
-                <h1>Create a Gistlog</h1>
-
-                <div class="bg-primary" style="padding: 1em;">
-                    <form method="post" action="/posts/create">
-                        <div class="form-group">
-                            <label for="gistUrl">Paste the URL to a Gist here to create a Gistlog:</label><br>
-                            <div class="row">
-                                <div class="col-md-7">
-                                    <input class="form-control" id="gistUrl" name="gistUrl" placeholder="Enter Gist URL">
-                                </div>
-                                <div class="col-md-4">
-                                    <button type="submit" class="btn btn-default">Create</button>
-                                </div>
-                            </div>
-                        </div>
-                    </form>
-                </div>
-
-                <br><br>
+<div class="container mx-auto mt-4">
+    <div class="text-center max-w-sm mx-auto">
+        <div class="mx-auto px-4 py-4">
+            <nav class="flex">
+                <img width="110px" class="mx-auto" src="{{ asset('/tailwind/img/logo.svg') }}">
```

Comment:
Why is the logo in the tailwind directory?

Reply:
I had a `/tailwind` directory during development, I forgot to make the change here once I moved all the images to the `/img` directory.